### DISCUSSION TITLE
The input box of the fyne is not compatible with the Chinese input method on the Mac system.

### DISCUSSION THREAD

**Discussion by EddieChan1993** [fyne-io/fyne#3798] [Q&A]
When using the content input box, there may be compatibility issues with the system's default input method. When the cursor is placed in the input box to enter text, the Chinese input method selection box may not be displayed. This issue is specific to Mac systems.

**Comment by andydotxyz** [comment]
This sounds like a bug report not a question.
It may be a duplicate as well - there are a number of Chinese / IME input tickets open.

**Reply by EddieChan1993** [reply]
总之，输入中文的时候，中文的选择菜单，不显示
